frontend/src/components/Options.vue

```diff
@@ -130,12 +130,11 @@ export default Vue.extend({
     this.hideRewards = localStorage.getItem('hideRewards') === 'true';
     this.hideWalletWarning = localStorage.getItem('hideWalletWarning') === 'true';
     this.showSkillInUsd = localStorage.getItem('showSkillInUsd') === 'true';
-    this.isBridgeEnabled = bridgeEnabled;
   },
 
   data() {
     return {
-      isBridgeEnabled: false,
+      isBridgeEnabled: bridgeEnabled,
```

Comment:
Is this necessary in user options?

Reply:
Nope, I did my flag as Bilbol did his in NFT Bridge, but there's no need to assign it to false and then actual value, so I just corrected it for NFT Bridge as well :)